test/gpdb/scripts/test_pitr_gpdb6.sh

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env bash
+set -exo pipefail
+
+PGBACKREST_TEST_DIR=/home/gpadmin/test_pgbackrest
+PGBACKREST_BIN=/usr/local/bin
+GPHOME=/usr/local/greenplum-db-devel
+
+echo "Starting up demo cluster..." > /dev/null
+source $GPHOME/greenplum_path.sh
+pushd gpdb_src/gpAux/gpdemo
+make create-demo-cluster WITH_MIRRORS=true
+source gpdemo-env.sh
+popd
+
+
+echo "Creating backup and log directories for pgbackrest..." > /dev/null
+TEST_NAME=$(basename "${0%.sh}")
+mkdir -p "$PGBACKREST_TEST_DIR/logs/$TEST_NAME"
+mkdir -p "$PGBACKREST_TEST_DIR/$TEST_NAME"
+
+DATADIR="${MASTER_DATA_DIRECTORY%*/*/*}"
+MASTER=${DATADIR}/qddir/demoDataDir-1
+PRIMARY1=${DATADIR}/dbfast1/demoDataDir0
+PRIMARY2=${DATADIR}/dbfast2/demoDataDir1
+PRIMARY3=${DATADIR}/dbfast3/demoDataDir2
+MIRROR1=${DATADIR}/dbfast_mirror1/demoDataDir0
+MIRROR2=${DATADIR}/dbfast_mirror2/demoDataDir1
+MIRROR3=${DATADIR}/dbfast_mirror3/demoDataDir2
+MASTER_PORT=6000
+PRIMARY1_PORT=6002
+PRIMARY2_PORT=6003
+PRIMARY3_PORT=6004
```

Comment:
These variables are used only once. What about placing the values in pgbackrest.conf without variables?
```
pg1-port=6000
```